### DISCUSSION TITLE
Relation widget, filter by previous field value_field

### DISCUSSION THREAD

**Discussion by secsonthebeach** [decaporg/decap-cms#7350] [Q&A]
Hello, I have been trying and failing at an attempt to use a previous relation widget “value_field” as the ‘filters: value:’ argument for a second relation widget. Should this be possible? Would this be similar to this question:
https://github.com/decaporg/decap-cms/discussions/7300

Specifically, I have a Hugo site where I am pulling in the Mitre ATT&CK data for reference and taxonomy tagging. My expected use case is an author creates new content using DecapCMS, and can apply taxonomy tags to their content, to reference the Mitre data.

I currently have hidden collections for all of the Mitre content, and am able to use the relation widget to provide authors with a drop-down for selecting what’s available. However, without some filtering, some of the available lists of content are very large and each of these could be applied to new content in ways that break the existing Mitre ATT&CK relations.

I would like to create a scenario where the author selects one of the 3 Mitre ‘domains’, which is then used as a filtering mechanism for available options of Mitre ‘tactics’. Similarly, The selected ‘tactic’ would be used in filtering the applicable ‘techniques’.

The existing Mitre content is such that the filtering/relations could be performed in either direction. Meaning: a selected domain can provide a list of nested tactics, or all of the available tactics could be filtered by their frontmatter which contains the domain tag.

I expected the following to work, but it has not:
```
- label: “Domain”
   name: “mitre-domain”
   widget: “relation”
   collection: “domains”
   search_fields: [“title”]
   value_field: ‘mitre-domains’
   display_fields: [“title”]

- label: “Tactic” 
   name: “mitre-tactic”
   widget: “relation”
   collection: “tactics”
   search_fields: [“title”]
   value_field: “mitre-tactics”
   display_fields: [“title”]
   filters:
      - field: mitre-domains
         values: [‘{{fields.mitre-domains}}’]  <— this doesn’t work
```
The domain widget properly applies the frontmatter, and manually applying the literal string properly filters the Tactics, but I am unable to pass the variable as an argument.

I am interested to hear any thoughts and ideas!

**Comment by martinjagodic** [comment]
@secsonthebeach this is not currently possible. You can rephrase this as a feature request and open an issue.
